securedrop_client/app.py

```diff
@@ -71,6 +72,21 @@ def configure_logging():
     sys.excepthook = excepthook
 
 
+def prevent_second_instance(app: QApplication) -> None:
+    # Null byte triggers abstract namespace
+    IDENTIFIER = '\0' + app.applicationName() + str(os.getuid())
+    ALREADY_BOUND_ERRNO = 98
+
+    app.instance_binding = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+    try:
+        app.instance_binding.bind(IDENTIFIER)
+    except OSError as e:
+        if e.errno == ALREADY_BOUND_ERRNO:
+            error_dialog = QMessageBox()
+            error_dialog.setText(app.applicationName() + ' is already running.')
+            sys.exit(error_dialog.exec())
```

Comment:
We might want an `else: raise` because if this goes bad, chances are something very bad is happening.

Reply:
Yeah, I have it on my local repo. Didn't bother to update because of #43, but it looks like that's merged now. I'll rebase, write some tests, and update.